docs/installing-solidity.rst

```diff
@@ -6,6 +6,17 @@
 Installing Solidity
 ###################
 
+Versioning
+==========
+
+Solidity versions follow `semantic versioning <https://semver.org>` and in addition to
+releases, **nightly development builds** are also made available.  The nightly builds
+are not guaranteed to be working and despite best efforts they might contain undocumented
+and/or broken changes. We recommend to use the latest release. All the method below will
```

Comment:
In order to ease development, we always had undocumented features. The rule is: If it is undocumented, it is subject to change without notice, but it will not affect documented behaviour. That way we can introduce new features and even try them out without having to wait for a release cycle or cherry-pick bugfix commits for patch releases.